core/scheduler/service/job_run_service.go

```diff
@@ -472,8 +472,9 @@ func (s *JobRunService) updateOperatorRun(ctx context.Context, event *scheduler.
 
 func (s *JobRunService) trackEvent(event *scheduler.Event) {
 	if event.Type.IsOfType(scheduler.EventCategorySLAMiss) {
-		s.l.Debug(fmt.Sprintf("received event: %v, jobName: %v , slaPayload: %#v",
-			event.Type, event.JobName, event.SLAObjectList))
+		s.l.Info(fmt.Sprintf("received %s event: %v, jobName: %v , slaPayload: %#v", scheduler.EventCategorySLAMiss, event.Type, event.JobName, event.SLAObjectList))
```

Comment:
what will be the difference of `scheduler.EventCategorySLAMiss` and `event.Type` in this case?

Reply:
they will be exact same, i.e. "sla_miss"

Reply:
let's change the log to `warn` instead of `info`. I don't think `error` suites the flow, though. and we can remove the `fmt.Sprintf` in this line.